.github/workflows/release_pypi.yml

```diff
@@ -25,6 +25,7 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: "3.10"
+        check-latest: true
```

Comment:
What is the expected behavior here? AFAIK `check-latest` checks if the latest python version is installed. But this will never be the case because the version is pinned to `3.10` just above.

Reply:
If it was just a one time error I am also fine with leaving as is.

Reply:
`3.10` fixes on the minor version, but still consumes patch version updates with `check-latest: true`. That's the effect.

Likewise, I'd also be ok with not having this change. This would eliminate one potential cause.

Reply:
I would keep it as is. There a no longer 3.10 versions released as current version is 3.11.

Reply:
Fair, that settles it.